app/ts/simulation/services/EtherScanAbiFetcher.ts

```diff
@@ -1,32 +1,55 @@
+import { ethers } from 'ethers'
 import { EtherscanGetABIResult, EtherscanSourceCodeResult } from '../../types/etherscan.js'
 import { EthereumAddress } from '../../types/wire-types.js'
-import { addressString } from '../../utils/bigint.js'
+import { addressString, checksummedAddress } from '../../utils/bigint.js'
 
-const EtherScanABIKey = 'PSW8C433Q667DVEX5BCRMGNAH9FSGFZ7Q8'
+const EtherscanABIKey = 'PSW8C433Q667DVEX5BCRMGNAH9FSGFZ7Q8'
```

Comment:
would be a good candidate to move to `constants.ts`

Reply:
I intended to move it for user to set, but yeah, it also should be in constant. I created issue about it